elasticsearch/exceptions.py

```diff
@@ -15,6 +15,8 @@
 #  specific language governing permissions and limitations
 #  under the License.
 
+from typing import Any, Dict, Type, Union
```

Comment:
Exceptions are being redone a bit with the move to elastic-transport hence the large amount of `type: ignore` for now.